### PR TITLE
minor improvements

### PR DESCRIPTION
**Pull request by veruu**

### CODE REVIEW

File: sktm/db.py [hunk 1]
```diff
@@ -482,22 +485,6 @@ def commit_tested(self, patches):
...
-
-        """
-        logging.debug("__commit_testrun: result=%s; buildid=%d",
-                      result, buildid)
-        self.cur.execute('INSERT INTO testrun(result_id, build_id) '
-                         'VALUES(?,?)',
-                         (result.value, buildid))
-        self.conn.commit()
-        return self.cur.lastrowid
-
```
Comment (spbnick) [inline sktm/db.py]: Yay, deleting code :smile:!

File: tests/test_db.py [hunk 2]
```diff
@@ -439,30 +412,29 @@ def test_unset_patchset_pending(self, mock_sql, mock_get_sourceid,
 
     @mock.patch('logging.debug')
     @mock.patch('sktm.db.SktDb._SktDb__get_baselineresult')
-    @mock.patch('sktm.db.SktDb._SktDb__commit_testrun')
     @mock.patch('sktm.db.SktDb._SktDb__get_repoid')
     @mock.patch('sktm.db.sqlite3')
     def test_update_baseline_new(self, mock_sql, mock_get_repoid,
-                                 mock_commit_testrun, mock_get_baselineresult,
-                                 mock_log):
+                                 mock_get_baselineresult, mock_log):
```
Comment (spbnick) [inline tests/test_db.py]: Could you please extract this and following changes into a separate commit, as they don't seem to be related?

Comment (veruu) [reply tests/test_db.py]: The `commit_testrun` method is being removed, so it can't be mocked and needs to be removed from the test signature too, and the fact that we now have one more DB interaction within the `update_baseline` method needs to be reflected in the tests so the commit doesn't cause a regression in the tests.

Comment (spbnick) [reply tests/test_db.py]: I see, thank you. Perhaps a note in the commit message would have helped me to understand what's going on and avoid unnecessary round-trip for explanations :)

Comment (veruu) [reply tests/test_db.py]: We have that "commit shouldn't break the build" rule, so I thought fixing the tests is implicit :) Will mention it in the future
